api/src/pcapi/routes/backoffice/templates/components/history/actions.html

```diff
@@ -27,12 +27,14 @@
             {% elif action.actionType.name in ["RULE_CREATED","RULE_DELETED","RULE_MODIFIED"] %}
               <p>{{ action.rule.name }}</p>
               {{ build_sub_rules_extra_data(action.extraData['sub_rules_info']) }}
+            {% elif action.actionType.name in ["LINK_VENUE_BANK_ACCOUNT_DEPRECATED", "LINK_VENUE_BANK_ACCOUNT_CREATED"] %}
+              Lieu: {{ links.build_venue_name_to_details_link(action.venue) }}
```

Comment:
Je suis très pointilleux :smile: 
```suggestion
              Lieu : {{ links.build_venue_name_to_details_link(action.venue) }}
```
(et donc aussi dans le test)

Reply:
done !